packages/protocol/contracts/governance/Governance.sol

```diff
@@ -46,9 +46,13 @@ contract Governance is
   }
 
   struct VoteRecord {
+    // obsolete
```

Comment:
would add the obsolete next to the variable rather than before. 

Can we also change the name of the variable to add something like `deprecated` at the beginning?

Reply:
I updated our tooling and renamed variable to depracated_[var_name]